packages/animation/src/extendAnimator/extendAnimator.test.tsx

```diff
@@ -16,11 +16,11 @@ test('Should extend actual animated component', () => {
     animator: AnimatorProvidedSettings
   }
```

Comment:
If you want to make all the tests to have the example component props, you can move this interface to be global in this file.